pymc/distributions/distribution.py

```diff
@@ -277,6 +314,14 @@ def __new__(
                 "for a standalone distribution."
             )
 
+        if "name" in kwargs:
```

Comment:
I think name couldn't be a kwarg before. Any reason to support it? If so, we need a test.

Reply:
`name` was a required argument so in principle `pm.Normal(name='x', mu=0., sigma=1.)` was valid beforehand. Since `name` is no longer an argument at all, I didn't want that example to break.

Reply:
My confusion I thought it was a positional only argument

Reply:
It was position only, but the code above is still valid pymc code.